lib/libc/stdlib/lib_mkstemp.c

```diff
@@ -65,6 +65,8 @@
 #include <semaphore.h>
```

Comment:
I think now this header file is not required.

Reply:
yes, it is not required. will remove it.